tests/functional/aws-node-sdk/test/object/get.js

```diff
@@ -658,13 +658,19 @@ describe('GET object', () => {
             it('If-None-Match & If-Modified-Since: returns NotModified when ' +
                 'Etag does not match and lastModified is greater',
                 done => {
-                    requestGet({
+                    const req = s3.getObject({
+                        Bucket: bucketName,
+                        Key: objectName,
                         IfNoneMatch: etagTrim,
                         IfModifiedSince: dateFromNow(-1),
                     }, err => {
                         checkError(err, 'NotModified');
                         done();
                     });
+                    req.on('httpHeaders', (code, headers, response, status) => {
+                        assert(!headers.hasOwnProperty('content-type'));
```

Comment:
Can we write something like
`assert(!headers.hasOwnProperty('content-type') || !headers.hasOwnProperty('Content-Type'));`

Reply:
Hmm, I know from the outputs the results have all lower case keys so not sure it is worth adding the other case.

Reply:
Hmm That is fine too. If the output has always lower case

Reply:
No need to check for case sensitiveness. Node.js converts all headers to lower case https://nodejs.org/api/http.html#http_message_headers
Also a check like `!headers['content-type']` is sufficient.